tasksJS/task1.js

```diff
@@ -0,0 +1,328 @@
+//----Strings----//
+
+//1 Remove String Spaces
+//https://www.codewars.com/kata/57eae20f5500ad98e50002c5
+
+function noSpace(x){
+    return x.split(" ").join("");
+}
+
+//2 Name Shuffler
+//https://www.codewars.com/kata/559ac78160f0be07c200005a 
+
+function nameShuffler(str){
+    return  str.split(" ").reverse().join(" ");
+}
+
+//3 Sum Mixed Array
+//https://www.codewars.com/kata/57eaeb9578748ff92a000009 
+
+function sumMix(x) {
+    let sum = 0;
+    for (let i = 0; i < x.length; i++) {
+        sum =sum + +x[i];
```

Comment:
`sum = sum + Number(x[i]);` is more readable

Reply:
as an option: 
sum += +x[i]
or
sum += Number(x[i])